### PR TITLE
[MM-30595] Make use of relative icon paths

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Downstream changes from https://github.com/mattermost/mattermost-plugin-apps/pull/197

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-30595

### CODE REVIEW

**Review by larkox — APPROVED** [review]

**Review by jfrerich — APPROVED** [review]
Love it. Thanks @hanzei!

**Comment by jfrerich** [comment]
I did a `DEV QA` for this PR nd verified the icons work using https://github.com/mattermost/mattermost-plugin-apps/pull/197.  Placing the the `Do Not Merge` until the apps PR is merged.

**Pull request merged**
